bin/cij_excavator

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+"""
+    CIJOE Metric Excavator
+"""
+import argparse
+import importlib.util
+import sys
+import os
+import cij.util
+import cij
+
+def trun_tcase_fpaths(root):
+    """Yields absolute paths to testcases below the given 'root'"""
+
+    for path, _, _ in os.walk(root):
+        if path.endswith(".sh"):
+            yield path
```

Comment:
I'm not sure I understand how this works -- is this supposed to be run with the trun root path? In that case, won't this yield paths to all hooks?

Reply:
But I think it's invoked with the trun root path, isn't it?

Reply:
Correct, starting at the test-result output dir walk it and yield testcase output-folders.

Reply:
It is to avoid loading the testrun e.g. trun.yml, as the extractor does not really need it, it just needs to find files of interest and emit metrics.